workflow/rules/functions.smk

```diff
@@ -17,7 +17,13 @@ def _get_container_metadata(refcon_name=None):
     else:
         refcon_md = [(key.split('_')[-1], metadata) for key, metadata in config.items() if key.startswith('metadata_')]
     if not refcon_md:
-        raise ValueError('No "metadata_" keys found in config. Did you forget to load a reference container config?')
+        base_image_name = [(key.split('_')[-1], metadata) for key, metadata in config.items() if key.startswith('metadata_')]
```

Comment:
if not `metadata_` keys are in the config (no config loaded), that list should be empty, and, consequently, the following lines won't reach the desired state